src/components/AllContents/index.js

```diff
@@ -0,0 +1,114 @@
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+
+import Title from '../Title';
+import InputForm from '../InputForm';
+import YetList from '../YetList';
+import DoneList from '../DoneList';
+
+const Index = () => {
+  const listText = 'list';
+  // 텍스트 이름과 변수 이름을 같게하면 좋을 것 같다는 피드백을 받았던 거 같은데
+  // list 를 다른 변수에 사용해 버렸음. 변수 이름 정하는 센스를 더 길러야 겠음
+
+  const localData = JSON.parse(localStorage.getItem(listText)) || [];
+  // localDate 를 처음에 가져와서 새로고침해도 데이터가 유지되게 설정
+
+  const [yetNum, setYetNum] = useState(0);
+  const [doneNum, setDoneNum] = useState(0);
+  const [contents, setContents] = useState('');
+  const [list, setList] = useState(localData);
+  const onChange = (e) => setContents(e.target.value);
+  const onSubmit = (e) => {
+    e.preventDefault();
+    const obj = {
+      contents,
+      id: Date.now(),
+      isDone: false,
+    };
```

Comment:
`contents`도 key-value 형태로 적어 주시는 게 좋을 것 같습니다

```suggestion
    const obj = {
      content: contents,
      id: Date.now(),
      isDone: false,
    };
```